pontoon/teams/urls.py

```diff
@@ -70,7 +73,9 @@
                             # Team contributors
                             path(
                                 "contributors/",
```

Comment:
[As per the docs](https://docs.djangoproject.com/en/5.0/topics/cache/#the-per-view-cache), if multiple URLs point at the same view, each URL will be cached separately. In this example, `/de/contributors/` and `/it/contributors/` will be cached separately, as you may expect. But once a particular URL has been requested, subsequent requests to that URL will use the cache.